test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -538,6 +549,13 @@ class test_uct_cm_sockaddr : public uct_test {
                      (TEST_CM_STATE_SERVER_CONNECTED  | TEST_CM_STATE_CLIENT_CONNECTED |
                       TEST_CM_STATE_CLIENT_GOT_REJECT | TEST_CM_STATE_CLIENT_GOT_ERROR));
 
+        deadline = ucs_get_time() + ucs_time_from_sec(DEFAULT_TIMEOUT_SEC) *
+                                    ucs::test_time_multiplier();
+
+        ucs_memory_cpu_load_fence();
+        while ((m_delayed_conn_reqs.size() == 0) && (ucs_get_time() < deadline)) {
```

Comment:
while should wait for counter (conn requests) and then load_fence() and access the queue